.travis.yml

```diff
@@ -35,6 +35,18 @@ matrix:
         - cmake . && make && sudo make install && cd ..
         - sudo cp .travis/config.hpp /usr/include/armadillo_bits/config.hpp
 
+    - os: linux
+      dist: xenial
```

Comment:
I'm wondering if it makes sense to use travis stages to distinguish the builds; not sure can we just add ``stage: Julia``

Reply:
I think you're right, but let me try that in a different PR since it would be a total change to how the Travis build works.

Reply:
Sure makes sense.